diff --git a/node_modules/canvas-size/dist/canvas-size.esm.js b/node_modules/canvas-size/dist/canvas-size.esm.js
index 5d14d79..71eec64 100644
--- a/node_modules/canvas-size/dist/canvas-size.esm.js
+++ b/node_modules/canvas-size/dist/canvas-size.esm.js
@@ -215,7 +215,7 @@ function handleMethod(settings) {
     }
 }
 
-const canvasSize = {
+export const canvasSize = {
     maxArea() {
         let options = arguments.length > 0 && arguments[0] !== undefined ? arguments[0] : {};
         const sizes = createSizesArray({
